spitfire/runtime/filters.py

```diff
@@ -37,7 +39,7 @@ def passthrough_filter(value):
 def escape_html(value, quote=True):
     """Replace special characters '&', '<' and '>' by SGML entities."""
     value = simple_str_filter(value)
-    if isinstance(value, basestring):
+    if isinstance(value, six.string_types):
```

Comment:
`simple_str_filter` always calls `str()`

```suggestion
    if isinstance(value, str):
```